repair_sale_order/__manifest__.py

```diff
@@ -0,0 +1,22 @@
+# Copyright 2022 ForgeFlow S.L. (https://forgeflow.com)
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+
+{
+    "name": "Repair To Sale Order",
+    "version": "14.0.1.0.0",
+    "author": "ForgeFlow, Odoo Community Association (OCA)",
+    "website": "https://github.com/OCA/manufacture",
+    "summary": "Repair To Sale Order",
+    "category": "Repair",
+    "depends": ["repair", "repair_type", "sale", "sales_team"],
```

Comment:
I think this should depend on repair_type and sale only, the others  would be implied dependencies